content/docs/specification/content.md

```diff
@@ -43,151 +43,193 @@ Further links about URL encoding:
+## Description
 
-**Version**  
+To redirect request from Docker CLI to a specific docker image or a docker registry you can use `dockerv2` type. The difference between a Docker CLI request and a normal request is that Docker CLI sends a couple of request with different paths to fetch some information like image tags, manifests and etc. about the image. To handle these requests we use docker's v2 API to fetch the data for each request.
 
-* Key: **v**
-* Mandatory
-* Permitted values: "txtv0"
-* Example: "v=txtv0"
+You can point to a docker registry or a specific image inside your record. With pointing to a registry you can have vanity urls for all the images on that registry instead of only an image.
 
-**Backend URL**  
+## Record Fields
 
-* Key: **to**
-* Mandatory
-* Permitted values: "absolute registry URL"
-* Note: "For non container traffic it will fallback to *website*, *www* or *redirect* config"
-* Example: "to=https://gcr.io/" *pass through namespaces, container and tag*
-* Example: "to=https://gcr.io/txtdirect/container" *pass through tag*
-* Example: "to=https://gcr.io/txtdirect/container:tag" *force specific tag*
+**Type**
 
-**Website Redirect URL**  
+- Key: **type**
+- Mandatory
+- Permitted values: "dockerv2"
+- Example: "type=dockerv2"
 
-* Key: **website**
-* Recommended
-* Default: Fallback to **www** or **redirect** config
-* Note: "Non container traffic redirect"
-* Permitted values: "absolute|relative URL"
-* Example: "to=https://about.txtdirect.org/docs/"
-* Example: "to=/docs/"
+**Version**
+
+- Key: **v**
+- Mandatory
+- Permitted values: "txtv0"
+- Example: "v=txtv0"
+
+**Backend URL**
+
+- Key: **to**
+- Mandatory
+- Permitted values: "absolute registry URL"
+- Note: "For non container traffic it will fallback to _website_, _www_ or _redirect_ config"
+- Example: "to=https://gcr.io/" _pass through namespaces, container and tag_
+- Example: "to=https://gcr.io/txtdirect/container" _pass through tag_
+- Example: "to=https://gcr.io/txtdirect/container:tag" _force specific tag_
+
+**Website Redirect URL**
+
+- Key: **website**
+- Recommended
+- Default: Fallback to **www** or **redirect** config
+- Note: "Non container traffic redirect"
+- Permitted values: "absolute|relative URL"
+- Example: "to=https://about.txtdirect.org/docs/"
+- Example: "to=/docs/"
 
 ---
+
 # Gometa Type
-**Type**  
 
-* Key: **type**
-* Mandatory
-* Permitted values: "gometa"
-* Example: "type=gometa"
+## Description
 
-**Version**  
+One of the features that `go get` command has is that you can point a webpage to a Go package using go-import meta tag. For more information about these tags please visit [Remote import paths](https://golang.org/cmd/go/#hdr-Remote_import_paths). Using `gometa` type you can point to a go package inside your records using `to=` field and we use that URI to generate a simple HTML page that only contains go-import and go-source **(Only for packages served on GitHub)** tags for `go get` to use it and find your package.
```

Comment:
This is explaining too much about go get and the go-import meta tag.
I would say something along the lines of: vanity url, control your import paths independent of your backend code repo (github/gitlab). easily switchable without impacting downstream users of your packages.
Using go-import meta tag under to implement this.

This part is great:
```
Using `gometa` type you can point to a go package inside your records using `to=` field and we use that URI to generate a simple HTML page that only contains go-import and go-source **(Only for packages served on GitHub)** tags for `go get` to use it and find your package.
```